Add Recursive Invocations Dropped documentation (#25672)
DataDog/documentation: content/en/serverless/guide/serverless_warnings.md

**Resolution:** Lambda functions approaching the maximum timeout limit of 15 minutes risk termination by the Lambda runtime. This could lead to slow or failed responses to incoming requests. Consider improving the performance of your Lambda function, using smaller lambdas in a Step Function, or moving your workload to a longer running environment like ECS Fargate.

> ### Recursive Invocations Dropped

Reviewer: ```suggestion
### Recursive invocations dropped
```

> Invocations in this function have a recursive loop, generally caused by recursive triggering between AWS entities (e.g., Lambda => SQS => Lambda). When this exceeds your maxReceiveCount (default 16), then it adds to this metric. More information can be found [here](15).

Reviewer: ```suggestion
Invocations in this function have a recursive loop, generally caused by recursive triggering between AWS entities (for example, Lambda -> SQS -> Lambda). When this exceeds your `maxReceiveCount` (default 16), then it adds to this metric. For more information, see [Use Lambda recursive loop detection to prevent infinite loops][15].
```

> **Resolution:** Find recursive calls in your AWS entities related to this function. Look for related entities such as [SQS, SNS, and S3](16).

Reviewer: ```suggestion
**Resolution:** Find recursive calls in your AWS entities related to this function. Look for related entities such as [SQS, SNS, and S3][16].
```
